app/src/app/components/contents/common/hooks/useGetPacketById.ts

```diff
@@ -5,12 +5,17 @@ import { PacketMetadata } from "../../../../../types";
 
 export const useGetPacketById = (packetId: string | undefined) => {
   const { data, isLoading, error } = useSWR<PacketMetadata>(
-    `${appConfig.apiUrl()}/packets/metadata/${packetId}`,
+    packetId ?
+      `${appConfig.apiUrl()}/packets/metadata/${packetId}` : null,
     (url: string) => fetcher({ url })
   );
 
+  const displayName = data?.custom?.orderly?.description?.display;
+
+  const packet = displayName ? { ...data, displayName } : data;
```

Comment:
there is displayName in the interface? whats that for?
.. also instead of spreading you can probs just do `{data, displayName: data?.custom?.orderly?.description?.display;` ... it will be undefined anyways If not there

Reply:
That change makes sense.

Yes, in the PacketMetadata interface, there is a displayName property. You added it in November 2023! And now we're using it for the first time, in three components, all of which use it to pass it as a prop to the PacketHeader.

Reply:
Actually no sorry you didn't add it, you just changed the indentation... it was added here https://github.com/mrc-ide/packit/commit/fc88d6393b336908f848cd63b1aa096371d42ad4